### PR TITLE
stratisd.spec: add dracut files

### PR DESCRIPTION
**Pull request by bgurney-rh**
Supersedes: #164

### CODE REVIEW

**Review by mulkieran — APPROVED** [review]

**Comment by bgurney-rh** [comment]
As tested on a new Fedora 34 VM with `dracut-053-2.fc34.x86_64`, this successfully builds and passes tests.

File: blackbox/stratisd.spec [hunk 1]
```diff
@@ -56,12 +58,27 @@ mv %{buildroot}%{_bindir}/stratisd %{buildroot}%{_libexecdir}/stratisd
 %{__install} -Dpm0644 -t %{buildroot}%{_mandir}/man8 docs/stratisd.8
 %{__install} -Dpm0644 -t %{buildroot}%{_udevrulesdir} udev/61-stratisd.rules
 %{__install} -Dpm0644 -t %{buildroot}%{_unitdir} systemd/stratisd.service
+%{__install} -Dpm0644 -t %{buildroot}%{dracutdir}/dracut.conf.d dracut/90-stratis.conf
+mkdir -p %{buildroot}%{dracutdir}/modules.d/90stratis
+# dracut provides the 90stratis/module-setup.sh file
```
Comment (mulkieran) [inline blackbox/stratisd.spec:63]: Please remove the comment and add the install of ```module_setup.sh```. There should be no conflict (on f34).

**Review by mulkieran — CHANGES_REQUESTED** [review]

File: blackbox/stratisd.spec (hunk 1 above)
Comment (bgurney-rh) [reply blackbox/stratisd.spec]: OK, I see it now.  Before, the `dracut` package had that file, and resulted in a conflict.  Now, `dracut` does not have that file, and `stratisd` must provide it.

**Review by mulkieran — CHANGES_REQUESTED** [review]
This looks correct to me.

Please close this and open a PR for final review.

**Pull request closed without merging**